analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AvoidLambdaExpressionInLoopsInBlazor.cs

```diff
@@ -77,5 +78,10 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
             builder.AddMarkupContent(14, "\r\n        Button");
             builder.CloseElement();
         }
+
+        foreach (var button in Buttons.OrderByDescending(x => x.Id))
+        {
+
```

Comment:
The empty line seems to suggest that there is code missing.
If no code is missing I would inline the braces and add a `// Compliant` comment, explaining why the scenario is compliant.